src/index.js

```diff
@@ -40,8 +38,31 @@ export default function onClickOutsideHOC(WrappedComponent, config) {
       return ref.getInstance ? ref.getInstance() : ref;
     }
 
-    // this is given meaning in componentDidMount/componentDidUpdate
-    __outsideClickHandler = null;
+    // this is given meaning in enableOnClickOutside
+    __internalOutsideClickHandler = null;
```

Comment:
wanted to split those 2 - `__internalOutsideClickHandler` and `__outsideClickHandler` but had not better names in my head 😄